modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Posts/Detail.cshtml

```diff
@@ -238,15 +238,15 @@
                                                             <i class="fa fa-reply" aria-hidden="true"></i> @L["Reply"]
                                                         </a>
                                                     }
-                                                    @if (await Authorization.IsGrantedAsync(BloggingPermissions.Comments.Delete) || (CurrentUser.Id == commentWithRepliesDto.Comment.CreatorId))
```

Comment:
Why did you remove `CurrentUser.Id == commentWithRepliesDto.Comment.CreatorId`?

Reply:
I forgot to merge the latest branch since the creator check was added into AuthorizationHandler 4 days ago. 😅